test/components/FileInput.spec.js

```diff
@@ -144,4 +150,32 @@ describe('components', () => {
       expect(tree).toMatchSnapshot();
     });
   });
+
+  describe('Metadata', () => {
+    const data = {
+      name: 'Cute puppies',
+      size: 1000,
+    };
+
+    it('should render metadata from default component', () => {
+      const { fileInput } = setup();
+      fileInput.setState({ value: data });
+
+      expect(fileInput.find('FileInputMetadata').length).toEqual(1);
```

Comment:
Just as a tip, you can just `expect(fileInput.find('FileInputMetadata')).toHaveLength(1);`.

(In jest's magic world you can do it with any object that has a length property -as far as I know).